### ISSUE TITLE
ModelContainer only opens specified EXPTYPE files

### ISSUE THREAD

**Issue by stscieisenhamer** [spacetelescope/jwst#786]
Add an argument to ModelContainer, `exptypes`, which takes a list of association `exptype` values to open from an association:
```
container = ModelContainter('an_asn.json', exptypes=['SCIENCE', 'PSF'])
```
The default would be `['SCIENCE']`

**Comment by nden** [comment]
@stscieisenhamer  Looking at the current state of the code, it looks like this may have been resolved already?

**Comment by jdavies-st** [comment]
Yes, this was resolved by #3857
